src/app/about/page.tsx

```diff
@@ -18,7 +18,7 @@ export default function Index() {
 
         <p>The idea for DATA emerged during a meeting on June 15, 2024, at Cole Park in Uptown Dallas, where approximately 15 people gathered to explore the possibility of forming a group. A second meeting on July 10, 2024, at the Metropolitan Building in Downtown Dallas saw over 50 attendees, during which interim co-chairs Tyler Wright and Connor Hulla were elected, and the organization's priorities were set. Our organization’s official name was chosen on July 25 after 137 people cast their votes.</p>
 
-        <p>As part of our advocacy efforts, Dallas transportation advocate Hexel Colorado started a <a href="https://www.change.org/p/protect-our-public-transit-say-no-to-dart-funding-cuts">petition on Change.org</a>, urging the DART Board of Directors to reject proposed funding cuts. Since its launch on July 16, the petition has garnered over 1,160 signatures and counting. As stated in the petition, we call upon the DART Board of Directors to reject any proposals to reduce the sales tax contributions from the thirteen member cities. Should the board vote on this matter, we implore every board member to stand against the cuts.</p>
+        <p>As part of our advocacy efforts, Dallas transportation advocate Hexel Colorado started a <a href="https://www.change.org/p/protect-our-public-transit-say-no-to-dart-funding-cuts">petition on Change.org</a>, urging the DART Board of Directors to reject proposed funding cuts. Since its launch on July 16, the petition garnered over 1,160 signatures and counting. The petition called upon the DART Board of Directors to reject any proposals to reduce the sales tax contributions from the thirteen member cities.</p>
```

Comment:
```suggestion
        <p>As part of our advocacy efforts, Dallas transportation advocate Hexel Colorado started a <a href="https://www.change.org/p/protect-our-public-transit-say-no-to-dart-funding-cuts">petition on Change.org</a>, urging the DART Board of Directors to reject proposed funding cuts. Since its launch on July 16, the petition garnered over 1,160 signatures in two weeks. The petition called upon the DART Board of Directors to reject any proposals to reduce the sales tax contributions from the thirteen member cities.</p>
```